pkg/get/get_test.go

```diff
@@ -7739,3 +7739,56 @@ func Test_DownloadLazyDocker(t *testing.T) {
 	}
 
 }
+
+func Test_DownloadKeploy(t *testing.T) {
+	tools := MakeTools()
+	name := "keploy"
+
+	tool := getTool(name, tools)
+
+	const toolVersion = "v2.2.0-alpha19"
```

Comment:
I'm kinda against the idea of using an alpha version here, could result some false negatives in the future since there is a higher chance that overtime they'll get rid of the alpha and beta releases, but keep the normal ones.

Reply:
right now, there are only alpha versions and only the versioning would change and not tag's. alpha-19 is outdated with alpha-22. I noticed in above test as well the versions were outdated so i went with most recent version at that time.